dev/io.openliberty.microprofile.config.2.0.internal_fat/fat/src/io/openliberty/microprofile/config/internal_fat/Config20ExceptionTests.java

```diff
@@ -49,7 +48,7 @@ public class Config20ExceptionTests extends FATServletClient {
     public static final String SERVER_NAME = "Config20ExceptionServer";
 
     @ClassRule
-    public static RepeatTests r = MicroProfileActions.repeat(SERVER_NAME, MicroProfileActions.LATEST, MicroProfileActions.MP50);
```

Comment:
Do we want these config tests to run against MP60 now?

Reply:
Perhaps but that wasn't the aim of this PR. Repeating Config FATs against MP6 would be a separate PR I think.

Reply:
Most of the config FATs have been updated for MP6 - I wasn't sure of the intention of these ones.